web/src/user_groups_settings_ui.js

```diff
@@ -129,7 +129,6 @@ export function add_group_to_table(group) {
         can_edit: user_group_edit.can_edit(group.id),
     });
 
-
```

Comment:
This line was added in the previous commit itself. We should remove it there only to avoid this change in this commit.

Reply:
Ohh, fixed this.